tests/test_controllers/MainConfig_test.py

```diff
@@ -51,7 +51,7 @@ def test_env_var_list_values_with_comma(self):
 
     yaml_config = """
 System:
-    FTS_MAINLOOP_DELAY: 600
+    FTS_MAINLOOP_DELAY: 100
```

Comment:
@naman108 One of the reasons that this setting was set to an oddball value was to insure that the value from the YAML config was actually applied. The default value for `MainLoopDelay` is 100 and setting the same value in the YAML config makes it impossible to determine if the YAML configuration was applied correctly or not.

Reply:
oh, I misunderstood, it was failing when I executed the tests so I figured it was a typo, is it passing on your machine?

Reply:
Yes it was. I can pull the latest master and test again to make sure. Do you have the failure that you can post here?

Reply:
On `master` which still has a value of 600:

```
❯ python -m pytest -s tests/test_controllers/MainConfig_test.py
/Users/hickey/.pyenv/versions/fts/lib/python3.10/site-packages/pep8.py:110: FutureWarning: Possible nested set at position 1
  EXTRANEOUS_WHITESPACE_REGEX = re.compile(r'[[({] | []}),;:]')
============================================================================== test session starts ==============================================================================
platform darwin -- Python 3.10.6, pytest-7.2.0, pluggy-1.0.0
rootdir: /Users/hickey/Projects/tak/fts
plugins: cov-4.0.0, pep8-1.0.6
collected 12 items

tests/test_controllers/MainConfig_test.py ............

============================================================================== 12 passed in 0.16s ===============================================================================
```